lib/jsdom/living/domparsing/serialization.js

```diff
@@ -20,20 +21,25 @@ function htmlSerialization(node) {
   return parse5.serialize(node, { treeAdapter });
 }
 
-module.exports.fragmentSerialization = (node, { requireWellFormed }) => {
+module.exports.fragmentSerialization = (node, { requireWellFormed, globalObject }) => {
```

Comment:
No&nbsp;need to&nbsp;[invoke&nbsp;`[[Get]]`](https://tc39.es/ecma262/#sec-ordinary-object-internal-methods-and-internal-slots-get-p-receiver) on&nbsp;`module`:
```suggestion
exports.fragmentSerialization = (node, { requireWellFormed, globalObject }) => {
```

Reply:
I know you have very particular preferences about how to use the `module` and `exports` objects, but I'd rather not change what exists in the code.